.travis.yml

```diff
@@ -3,3 +3,4 @@ elixir:
   - 1.3.4
   - 1.4.5
   - 1.5.1
+  - 1.9.0
```

Comment:
I am not sure if this is required as part of this changeset. Some of the older versions, as well as the recently released 1.10.0 version, is missing from this list.

Reply:
I'll review the versions here when I have some time, meanwhile this addition is great 👍